km/km_init_guest.c

```diff
@@ -329,14 +329,23 @@ int km_clone(km_vcpu_t* vcpu,
    return km_vcpu_get_tid(new_vcpu);
 }
 
-long km_clone3(km_vcpu_t* vcpu, struct clone_args* cl_args)
+long km_clone3(km_vcpu_t* vcpu, struct clone_args* cl_args, size_t cl_args_len)
 {
-   if (cl_args->set_tid_size != 0 || cl_args->cgroup != 0) {
+#ifdef CLONE_ARGS_SIZE_VER2
```

Comment:
What about CLONE_ARGS_SIZE_VER0? That exists.

Reply:
Let me rephrase all V0 members are used.